omnigibson/object_states/object_state_base.py

```diff
@@ -47,6 +46,7 @@ def __init__(self, obj):
         self._cache = None
         self._changed = None
         self._simulator = None
+        self._last_t_updated = -1               # Last timestep when this state was updated
```

Comment:
sorry I am not sure I understand why we need this

Reply:
resolved?